saltgui/static/scripts/routes/pillarsminion.js

```diff
@@ -43,35 +43,60 @@ class PillarsMinionRoute extends PageRoute {
 
     if(!pillars) return;
 
+    // collect the public pillars and compile their regexps
+    let publicPillarsText = localStorage.getItem("public_pillars");
+    if(!publicPillarsText || publicPillarsText === "undefined") publicPillarsText = "[]";
+    let public_pillars = JSON.parse(publicPillarsText);
+    if(!Array.isArray(public_pillars)) public_pillars = [ ];
+    for(let i = 0; i < public_pillars.length; i++) {
+      try {
+        public_pillars[i] = new RegExp(public_pillars);
```

Comment:
This should be:
```
public_pillars[i] = new RegExp(public_pillars[i]);
```

Reply:
wow, amazing that all my tests went well...